### PR TITLE
feat(init): success message

### PR DESCRIPTION
**Pull request by denkristoffer**
Shows a success message after going through the `init` flow.

Screenshot 2022-08-18 at 14 51 01

### CODE REVIEW

File: lib/cmds/init/success.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { highlight } from 'cli-highlight'
```
Comment (mshaaban0) [inline lib/cmds/init/success.ts:1]: I'd like to see `chalk` imported here :) we neeeeeed colors to differentiate between output sections!
Try not to over do it though, let's keep it subtle. 🙏🏼 ❤️

File: lib/cmds/init.ts [hunk 2]
```diff
@@ -76,6 +77,15 @@ export const init = async () => {
       })
     }
   }
+
+  // something something, get the flow type
+  // one of 'Contentful.js', 'GraphQL', 'REST API'
+  const connectionType = 'GraphQL'
```
Comment (mshaaban0) [inline lib/cmds/init.ts:83]: Yup, we need another prompt here to choose one of ....

**Review by mshaaban0 — COMMENTED** [review]
I can't seem to get any of the success output when going through the init command 🤔
Screenshot 2022-08-19 at 16 55 57

File: lib/cmds/init/success.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  const client = contentful.createClient({
+    space: "${spaceId}",
+    accessToken: "${accessToken}"
+  });
+  client
+    .getEntry("${entryId}")
+    .then(entry => console.log(entry))
+    .catch(err => console.log(err));
+    `
+    ),
```
Comment (mshaaban0) [inline lib/cmds/init/success.ts:33]: second parameter accepts `language: 'javascript'`, it should help with coloring these 

```js
highlight(graphQlExample, {
  language: "JavaScript",
})
```

File: lib/cmds/init.ts (hunk 2 above)
Comment (denkristoffer) [reply lib/cmds/init.ts:83]: Wouldn't that come from previously in the flow?

**Review by mshaaban0 — APPROVED** [review]
Let's merge this and I'll follow up with putting the steps together 👍🏼

**Pull request merged**